src/openrct2/CmdlineSprite.cpp

```diff
@@ -246,7 +246,7 @@ static bool sprite_file_export(rct_g1_element* spriteHeader, const char* outPath
 
 static bool sprite_file_import(
```

Comment:
Can't we return a std::optional ImportResult? Then you can get rid of all the out parameters.